validators/performance/aiperf-bench.Dockerfile

```
@@ -19,28 +19,63 @@
-# running as a dedicated non-root user hardens the image for air-gap /
-# multi-tenant deployments.
-#
-# Upgrade pip first so the install uses a pip with current CVE fixes
-# (the base image ships an older pip pinned to its release date).
+ARG AIPERF_VERSION
+
+# build-essential supplies gcc/g++/make for source builds (crick has no
+# linux/arm64 wheel; pyzmq/uvloop have no wheel on a Python without cp3XX
+# wheels). pyzmq additionally drives its build through cmake, which pip
+# installs into its isolated build environment automatically.
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)